src/osd/PrimaryLogPG.cc

```diff
@@ -12739,6 +12749,9 @@ bool PrimaryLogPG::agent_choose_mode(bool restart, OpRequestRef op)
     num_user_objects = 0;
 
   uint64_t num_user_bytes = info.stats.stats.sum.num_bytes;
+  // TODO should go depend on osd->store
```

Comment:
I'm not sure how to do that, though.  These stats are 'logical' stats that describe the user data we store, not how efficiently we store them.  The stats can't/shouldn't change when an instance of the PG moves to a different OSD and/or backend.  And what would happen if one replica were FileStore and one BlueStore?